app/timetables/tests/test_models.py

```diff
@@ -192,3 +192,20 @@ def test_cycle_length_and_current_cycle_day_of_negative_value_cant_be_saved(self
         )
 
         self.assertRaises(ValidationError, timetable.save)
+
+
+class DishTest(TestCase):
+    """Tests the Dish model."""
+
+    def setUp(self):
+        Dish.objects.create(
+            name='eba',
+            description='eba and garri. Make eba for your wedding day.',
+            date_created=datetime.strptime('5:7:9', '%H:%M:%S'),
+            date_modified=datetime.strptime('10:7:9', '%H:%M:%S')
```

Comment:
@andela-joyebanji the `'%H:%M:%S'` on the date_created and date_modified should be changed to a date format, same as the date. An example is this ```date_created=datetime.strptime('05 07 2016', '%d %m %Y'),
            date_modified=datetime.strptime('06 08 2016', '%d %m %Y')```